### PR TITLE
Add helper functions to map_util.js, slight interface fix to reduce the number of ways we pass latlongs around

### PR DESCRIPTION
**Pull request by enkait**
#37 The helpers will be used for fast travel and for showing a pin in the center of the player's locations on the final victory screen.

### CODE REVIEW

**Comment by enkait** [comment]
Note: I tested this by trying the game in both KML and regular modes, and it seemed to still work.

File: src/maps_util.js [hunk 1]
```diff
@@ -28,21 +31,84 @@ class GoogleMapsWrapper {
...
+		// (lng, lat) is the turf coordinate format.
+    const num_attempts = 10;
+    let panorama = null;
+    for (let attempt = 0; attempt < num_attempts; attempt++) {
+      let fraction = (num_attempts - attempt) / num_attempts;
+      const jump_point_turf = destination([point.lng, point.lat], distance_km * fraction, bearing_deg).geometry.coordinates;
+      // We first attempt to jump by distance_km, searching in a radius of 0.5 * distance_km.
+      // If that fails we attempt to back off each time by 1/num_attempts of distance_km.
+      // The radius gets progressively narrower to maintain the same distance-to-search radius proportion.
+      const radius_m = distance_km * 0.5 * fraction * 1000;
```
Comment (anula) [inline src/maps_util.js:79]: The radius seems quite high. This means that when we try travelling eg. north, we end up exactly to north-east.
Was it intended?

File: src/maps_util.js [hunk 2]
```diff
@@ -28,21 +31,84 @@ class GoogleMapsWrapper {
...
+  // elements.
+  centerPoint(input_points) {
+    let result = {lat: 0, lng: 0};
+    for (const point of input_points) {
+      result.lat += point.lat;
+      result.lng += point.lng;
+    }
+    result.lat /= input_points.length;
+    result.lng /= input_points.length;
+    return {lat: result.lat, lng: result.lng};
```
Comment (anula) [inline src/maps_util.js:111]: Why not just return result?

**Review by anula — APPROVED** [review]
LGTM, though pls rerun lint.

File: src/maps_util.js (hunk 2 above)
Comment (enkait) [reply src/maps_util.js]: Done.

File: src/maps_util.js (hunk 1 above)
Comment (enkait) [reply src/maps_util.js]: Change to 0.1 instead of 0.5. Thanks!

**Pull request merged**